### PR TITLE
Digest signed package fixes

### PR DESCRIPTION
**Pull request by xor-gate**
Re-enables package signing, but doesn't work yet.

### CODE REVIEW

File: lib/targzip/targz.go [hunk 1]
```diff
@@ -171,6 +171,19 @@ func (t *TarGzip) Name() string {
 	return t.fileName
 }
 
+func (t *TarGzip) Size() int64 {
```
Comment (rikvdh) [inline lib/targzip/targz.go]: You can just use `fi, err := os.Stat(t.Name())` instead of opening and closing the file.

File: digest_test.go [hunk 2]
```diff
@@ -5,7 +5,7 @@
 package debpkg
 
 import (
-	"fmt"
+//	"fmt"
```
Comment (rikvdh) [inline digest_test.go]: if you dont use fmt, just remove it.

**Comment by rikvdh** [comment]
Travis an Appveyor are failing furthermore LGTM

**Comment by xor-gate** [comment]
Appveyor and travis have been fixed. Only appveyor tests now with race detector on, because these workers are faster (travis takes approx 220sec, while appveyor 100sec).

**Comment by rikvdh** [comment]
You're the boss. I think data-race detection should be on by default everywhere and disable it when you can't wait. CI results is never real-time needed.

**Pull request merged**